apps/frontend/src/store/app_info.ts

```diff
@@ -54,7 +55,7 @@ export class AppInfo extends VuexModule implements IAppInfoState {
 
   @Action
   public async CheckForUpdates() {
-    if (this.checkedForUpdates === false) {
+    if (!ServerModule.disableUpdateCheck && this.checkedForUpdates === false) {
```

Comment:
Please move this into its own guard clause where we can set 'SET_VERSION' to the deployed instance's version and 'SET_CHECKED_FOR_UPDATES' to true.